spinalcordtoolbox/utils/__init__.py

```diff
@@ -1,4 +0,0 @@
-from .fs import *  # noqa: F401, F403
-from .sys import *  # noqa: F401, F403
-from .sys import __version__, __sct_dir__, __data_dir__, __deepseg_dir__  # noqa: F401
-from .shell import *  # noqa: F401, F403
```

Comment:
Based on the error message from the tests, it seems the issue is that the script was run in a way that skips `__init__.py` files, but that there are remaining utils imports in the package-level `__init__.py`?

I'm happy to push a fix for this, too!

Reply:
Yep, I forgot to consider other `__init__.py` files, and the CI scripts. Sorry!

Reply:
Hah. I wrote in my extended commit message:

> I couldn't find any instances of `from spinalcordtoolbox import __` in the project. I presume this isn't actually used? But perhaps it should be kept...

I was wrong!

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/runner/work/spinalcordtoolbox/spinalcordtoolbox/spinalcordtoolbox/deepseg/models.py", line 298, in install_default_models
    install_model(name_model)
  File "/home/runner/work/spinalcordtoolbox/spinalcordtoolbox/spinalcordtoolbox/deepseg/models.py", line 281, in install_model
    download.install_data(model_urls, folder(name_model))
  File "/home/runner/work/spinalcordtoolbox/spinalcordtoolbox/spinalcordtoolbox/deepseg/models.py", line 266, in folder
    return os.path.join(sct.__deepseg_dir__, name_model)
AttributeError: module 'spinalcordtoolbox' has no attribute '__deepseg_dir__'
```

Curse you, `import spinalcordtoolbox as sct`!

Reply:
Oh woof, I was especially wrong. I don't know what I thought I had searched, but `from spinalcordtoolbox import __` is a lot more prevalent than I thought. My bad...